index.js

```diff
@@ -0,0 +1,70 @@
+const TittleInput = document.querySelector('.title_input');
+const AuthorInput = document.querySelector('.author_input');
+const BtnAdd = document.querySelector('.btn_add');
+const BooksUl = document.querySelector('.books_list');
+
+function DeleteBook() {
+  const DeleteBtn = document.createElement('button');
+  DeleteBtn.textContent = 'Remove';
+  DeleteBtn.addEventListener('click', (e) => {
+    const item = e.target.parentElement;
+    BooksUl.removeChild(item);
+  });
+  return DeleteBtn;
+}
+
+BtnAdd.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const tittleo = TittleInput.value;
+  const authoro = AuthorInput.value;
+
+  const book = document.createElement('li');
+  const tittle = document.createElement('p');
+  const author = document.createElement('p');
+  const line = document.createElement('hr');
+
+  tittle.textContent = tittleo;
+  author.textContent = authoro;
+
+  BooksUl.append(book);
+  book.append(tittle);
+  book.append(author);
+  book.append(DeleteBook());
+  book.append(line);
+
+  TittleInput.value = '';
+  AuthorInput.value = '';
+});
+
+// Capture the form and form elements
+const author = document.getElementById('author_input');
+const title = document.getElementById('title_input');
+
+// Get data from the javascript object
+function getFormData() {
+  const userData = localStorage.getItem('userData');
+  if (userData !== null) {
+    const userDataObj = JSON.parse(userData);
+    AuthorInput.value = userDataObj.name;
+    TittleInput.value = userDataObj.text;
+  }
+}
+
+// Set data in the javascript object
+function setFormData() {
+  // Create a JavaScript object with the captured values
+  const contactInfo = {
+    name: author.value,
+    text: title.value,
+  };
+  // Save the object in the localStorage
+  localStorage.setItem('userData', JSON.stringify(contactInfo));
+}
```

Comment:
- [ ] As the previous reviewer pointed out, the data does not persist properly and does not display on reload. From my observation, you are 
  - not fetching the data from the localstorage on page load
  - completely replacing the data in the localstorage with the new one instead of appending to it.